fetch.bs

```diff
@@ -4391,6 +4394,14 @@ steps:
 
       <p class=note>This covers the case of <var>response</var> being a <a>network error</a>.
 
+     <li><p>Let <a for=response>body info</a>'s <a for="response body info">content type</a> be the
+     empty string if <var>response</var>'s <a for=response>type</a> is "<code>opaque</code>" or
+     <var>fetchParams</var>'s <a for="fetch params">request</a>'s <a for=request>mode</a> is
+     `<code>navigate</code>` and <var>response</var>'s <a for=response>URL</a>'s
+     <a for=url>origin</a> is not <a>same origin</a> with <var>request</var>'s
+     <a for=request>origin</a>, and result of <a for="header list">extracting a MIME type</a> from
+     <var>response</var>'s <a for=response>header list</a> otherwise.
```

Comment:
I have similar concerns here to your other PR about the same origin check and redirects, and also how to word otherwise cases.

Also, when you can only "let" a variable, you need to "set" a field ("to" something).